integration-adapter/src/test/java/com/rln/CommonBaseTest.java

```diff
@@ -225,4 +230,20 @@ protected void eventually(Runnable code) throws InterruptedException {
             }
         }
     }
+
+    private static IBANs toIBANs(Step step) {
```

Comment:
Seems like we have this code already. I'd move that code to either a commonly accessibly utility function, or maybe into `Step`, although the latter is a bit questionable.

Reply:
I introduced a Utility.java